conceptnet5/db/connection.py

```diff
@@ -8,16 +8,15 @@
 _CONNECTIONS = {}
 
 
-def get_db_connection(dbname=None, building=False):
+def get_db_connection(dbname=None, building=None):
```

Comment:
It looks to me like the only place we call it now with `building` arg is `load_data` in `conceptnet5/conceptnet5/db/cli.py`. I think it would make sense to remove that argument there.

Reply:
Good idea.